### ISSUE TITLE
Scratch will support Sprite-deletion-confirmation natively

### ISSUE THREAD

**Issue by WorldLanguages** [ScratchAddons/ScratchAddons#7869]
https://github.com/scratchfoundation/scratch-gui/pull/9700/files

**Comment by CST1229** [comment]
also, Scratch's UI officially using the term "asset"
![image](https://github.com/user-attachments/assets/2e2a6a00-e98f-4343-8909-87745f49bb7c)

**Comment by DNin01** [comment]
> also, Scratch's UI officially using the term "asset"

Whether sprites are assets or not is debatable lol

**Comment by DNin01** [comment]
We should probably disable this addon once Scratch rolls out this feature. Here's a possible notice to tell users about this:
> This addon has been disabled - Scratch now has this feature natively! 👏

**Comment by Samq64** [comment]
![screenshot-2024-11-02-16-29-55](https://github.com/user-attachments/assets/257ea0c0-ba67-494c-b5bc-8b0cdc4247f7)

Only for sprites on [scratch-gui](https://scratchfoundation.github.io/scratch-gui/) so far, not costumes or sounds.

**Comment by DNin01** [comment]
Currently, combined with the addon, clicking the delete button will first pop up Scratch's new deletion confirmation, and then when clicking "yes", Scratch Addons will pop up a second deletion confirmation dialog. Confirming both of them will delete the sprite.

That's kind of what I expected. Nothing catastrophic, so we could either deprecate this addon with a minor release or wait until v1.40 and in the meantime hopefully users will realize to just disable the addon themselves.

**Comment by mxmou** [comment]
Editor dark mode currently breaks the popup:
![image](https://github.com/user-attachments/assets/cd081747-93e2-449b-9289-035900bc1bde)
I'll make a PR to fix that.

> Currently, combined with the addon, clicking the delete button will first pop up Scratch's new deletion confirmation, and then when clicking "yes", Scratch Addons will pop up a second deletion confirmation dialog. Confirming both of them will delete the sprite.

Ideally the addon would check if Scratch's confirmation popup is visible and not show a second confirmation in that case.

**Comment by Samq64** [comment]
This has been pushed to the main site now.

**Comment by medlans** [comment]
I've seen people state that they've wanted to disable this (not just me). Also, for sprites, I already have the delete button hidden behind right click, so I especially don't really want to have the confirmation there

**Comment by WorldLanguages** [comment]
Yes, we will remove the addon and could create the opposite addon which removes the confirmation (dangerous tag)
